### DISCUSSION TITLE
Compatibility issue when build service in AWS

### DISCUSSION THREAD

**Discussion by 6vvvvvv** [react-hook-form/react-hook-form#7497] [General] [answered]
Hi, everyone,

I am trying to build my project on AWS and a warning shows up:

**[WARNING]: error react-hook-form@7.22.5: The engine "node" is incompatible with this module. Expected version ">=12.22.0". Got "12.21.0"**

I can't upgrade the node version on AWS so I have to downgrade the version of react-hook-form. Can somebody recommend a compatible version of react-hook-form for me please?

Thanks.

**Comment by bluebill1049** [comment, marked as answer]
I think you can roll a few minor versions back when we do not have the expected version.

**Reply by 6vvvvvv** [reply]
Ok. Actually I can make a roll back on my side but the build job is on other people which turns it a little complicated in this situation. That's why I came here for this answer. Thanks anyway.
